bot/cogs/modlog.py

```diff
@@ -609,7 +608,8 @@ def ignore(self, event: Event, *items: int):
         )
 
     async def on_message_edit(self, before: Message, after: Message):
-        if before.guild.id != GuildConstant.id or before.channel.id in GuildConstant.ignored or before.author.bot:
+        if not before.guild or before.guild.id != GuildConstant.id \
+                or before.channel.id in GuildConstant.ignored or before.author.bot:
```

Comment:
The style we use for splitting long if statements is

```py
if (
    not before.guild
    or before.guild.id != GuildConstant.id
    or before.channel.id in GuildConstant.ignored 
    or before.author.bot
):
```

Admittedly, that would make it quite ugly.

Reply:
Yeah, I disliked the backslash here too, but wasn't sure how to write it more elegantly... Gonna change both occurrences.